sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/models/TopicDescription.java

```diff
@@ -100,6 +100,11 @@ private AuthorizationRulesWrapper(
                 @JacksonXmlProperty(localName = "AuthorizationRule") List<AuthorizationRuleImpl> items) {
             this.items = items;
         }
+        
+        @JsonCreator
+        private AuthorizationRulesWrapper() {
```

Comment:
@suxlv2 AdminClient.getTopic(), createSubscription() - is working with 7.7.0 version. Can you eloborate your code some more so that we can figure why you are running into this issue?

Reply:
Hi @ki1729 
The case described in this pull-request is currently a case of actual MS support ticket.
Company I work for suffered from issue, where app deployed in Azure AppService can't communicate to MS ASB due to the code problem we've "fixed" by adding the noArg constructor. Without it all we have is "deserialization error" as XML (response of SDK's internal call to getTopicWithResponse()) coming from ASB doesn't have the <AuthorizationRules> block (that we know from the exception stacktrace thrown by objectMapper). I don't have the updates from the MS ASB support team working on our case, but what we know is that our code fix is working. Of course it's up to you to decide.. if it's needed of not in public version.

Reply:
Still waiting for resolution.